### PR TITLE
New package: mcfly-0.8.3

### PR DESCRIPTION
**Pull request by tommyalatalo**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (amd64)

### CODE REVIEW

**Comment by altosys** [comment]
@classabbyamp how long is the expected wait to get a review for a PR like this? Do I need to somehow flag this as ready?

File: srcpkgs/mcfly/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+changelog="https://github.com/cantino/mcfly/blob/master/CHANGELOG.txt"
+distfiles="https://github.com/cantino/mcfly/archive/refs/tags/v${version}.tar.gz"
+checksum=727fc98b7291cc5b79c90a48d2e4460bc71550f221be8d2dad2377580f9b2d72
+
+post_install() {
+	vlicense LICENSE
+	vdoc README.md
+	vcompletion mcfly.fish fish
+	vcompletion mcfly.zsh zsh
+	vcompletion mcfly.bash bash
```
[the remark this answers is not in the document]
Comment (altosys) [reply srcpkgs/mcfly/template:19]: Hmm, you're right, those seem to be init scripts which I don't think are needed in the template now that you mention it, I'll just leave them out. The init command for the user's bashrc etc, `eval "$(mcfly init bash)"` should be left to the user to add manually, right?

[the remark this answers is not in the document]
Comment (tranzystorekk) [reply srcpkgs/mcfly/template:19]: Yes, exactly 😄

File: srcpkgs/mcfly/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'mcfly'
+pkgname=mcfly
+version=0.8.1
+revision=1
+build_style=cargo
+short_desc="Intelligent shell history search (ctrl-r) replacement"
+maintainer="Tommy Alatalo <tommy@alatalo.io>"
+license="MIT"
+homepage="https://github.com/cantino/mcfly"
+changelog="https://github.com/cantino/mcfly/blob/master/CHANGELOG.txt"
```
Comment (tranzystorekk) [inline srcpkgs/mcfly/template:10]:
```suggestion
changelog="https://raw.githubusercontent.com/cantino/mcfly/master/CHANGELOG.txt"
```

Comment (altosys) [reply srcpkgs/mcfly/template]: Pushed this change

**Comment by altosys** [comment]
> Almost LGTM,
> 
> I tested this package successfully, but we should probably add
> 
> ```shell
> configure_args="--no-default-features"
> hostmakedepends="pkg-config"
> makedepends="sqlite-devel"
> ```
> 
> to get it to link system libsqlite.

Added them

**Comment by altosys** [comment]
Pushed an update to 0.8.3 for this as the project released new versions late yesterday.

**Comment by altosys** [comment]
@tranzystorekk what is the procedure from here once the PR has been marked as approved?

**Comment by tranzystorekk** [comment]
My approval is only advisory for the core maintainers that this PR was tested by another person.

You can ask on irc from time to time to see if any core member is interested in merging but nothing is guaranteed.

The CONTRIBUTING.md (intentionally) vaguely states that a good candidate for template maintainership should have a record of smaller contributions, like package updates or template fixes.

Ultimately the core members carry the major part of responsibility for packages, which is why they have the final say in merges.

**Comment by altosys** [comment]
> My approval is only advisory for the core maintainers that this PR was tested by another person.
> 
> You can ask on irc from time to time to see if any core member is interested in merging but nothing is guaranteed.
> 
> The CONTRIBUTING.md (intentionally) vaguely states that a good candidate for template maintainership should have a record of smaller contributions, like package updates or template fixes.
> 
> Ultimately the core members carry the major part of responsibility for packages, which is why they have the final say in merges.

Ah great, thanks for the clarification. :)

**Comment by tunalad** [comment]
can you guys reopen this PR? MyFly is like the only good shell history manager that isn't wack

**Pull request closed without merging**